pkg/service/rtcservice.go

```diff
@@ -180,6 +184,26 @@ func (s *RTCService) validateInternal(r *http.Request) (livekit.RoomName, routin
 	}
 	SetRoomConfiguration(createRequest, claims.GetRoomConfiguration())
 
+	if attributesStr != "" {
+		data, err := base64.URLEncoding.DecodeString(attributesStr)
+		if err != nil {
+			return "", pi, http.StatusBadRequest, errors.New("failed to parse attributes")
+		}
+		var attrs map[string]string
+		if err := json.Unmarshal(data, &attrs); err != nil {
+			return "", pi, http.StatusBadRequest, errors.New("failed to parse attributes")
+		}
+		if len(attrs) != 0 && claims.Attributes == nil {
+			claims.Attributes = make(map[string]string)
+		}
+		for k, v := range attrs {
+			if v == "" {
+				continue // do not allow deleting existing attributes
+			}
+			claims.Attributes[k] = v
+		}
+	}
+
```

Comment:
maybe not return errors if decoding/unmarshalling fails just to be defensive, i. e. just log and ignore and if any step errors?

Reply:
Done! But instead of ignoring the error completely, it will now return error on `/rtc/validate` but proceed with a log on regular `/rtc`. Does it sound reasonable?